networkQuality.go

```diff
@@ -303,31 +355,35 @@ func main() {
 	 */
 	generate_lgd := func() lgc.LoadGeneratingConnection {
 		return &lgc.LoadGeneratingConnectionDownload{
-			Path:      config.Urls.LargeUrl,
-			Host:      config.Urls.LargeUrlHost,
-			KeyLogger: sslKeyFileConcurrentWriter,
+			URL:                config.Urls.LargeUrl,
+			KeyLogger:          sslKeyFileConcurrentWriter,
+			ConnectToAddr:      config.ConnectToAddr,
+			InsecureSkipVerify: config.InsecureSkipVerify,
```

Comment:
I would prefer that we continue to refer here to `*insecureSkipVerify` to make it clear that the value comes from the command line and is not somehow overridden by the data that comes back from the server's configuration.

Reply:
Done!